frechet_audio_distance/fad.py

```diff
@@ -7,20 +7,23 @@
 """
 import os
 import numpy as np
+import resampy
+import soundfile as sf
 import torch
-from torch import nn
+import laion_clap
+
+from multiprocessing.dummy import Pool as ThreadPool
 from scipy import linalg
+from torch import nn
 from tqdm import tqdm
-import soundfile as sf
-import resampy
-from multiprocessing.dummy import Pool as ThreadPool
-from .models.pann import Cnn14_8k, Cnn14_16k, Cnn14
 
+from .models.pann import Cnn14, Cnn14_8k, Cnn14_16k
 
-SAMPLE_RATE = 16000
+# SAMPLE_RATE = 16000
 
 
-def load_audio_task(fname, dtype="float32"):
+def load_audio_task(fname, sample_rate, dtype="float32"):
+    # print("LOAD AUDIO TASK")

```

Comment:
This line can be removed too.

Reply:
Yes, I forgot to remove the debugging print